### PR TITLE
doc: post 1.9.0 doc update

### PR DESCRIPTION
**Pull request by b-gent**
Post-release updates to docs.

Signed-off-by: Bartosz Gentkowski <bartosz.gentkowski@nordicsemi.no>

### CODE REVIEW

**Comment by b-gent** [comment]
Things to do as soon as #6707  and #6812 are merged:

- add known issues back to index
- remove orphan from known issues
- add 1.9.99 to versions.json
- update SES links in links.txt
- add changelog to release notes index

**Comment by greg-fer** [comment]
> add changelog to release notes index

Also, @b-gent , check that the changelog is cleaned up and/or contains only entries for 1.9.99.

**Comment by b-gent** [comment]
> 

Changelog is being cleaned by Uma in the release notes PR.

File: doc/nrf/release_notes.rst [hunk 1]
```diff
@@ -12,13 +12,15 @@ This page is included only in the latest documentation, because it might contain
    A "99" at the end of the version number of this documentation indicates continuous updates on the main branch since the previous major.minor release.
    When looking at this latest documentation, be aware of the following aspect:
```
Comment (umapraseeda) [inline doc/nrf/release_notes.rst:13]:
```suggestion
   When looking at this latest documentation, be aware of the following aspects:
```

File: doc/nrf/release_notes.rst [hunk 2]
```diff
@@ -12,13 +12,15 @@ This page is included only in the latest documentation, because it might contain
...
 
+   * Changes between releases are tracked on the :ref:`ncs_release_notes_changelog` page, but the main branch might contain additional changes that are not listed on that page.
    * The release note pages that are available in the latest documentation might differ slightly from the release notes that were included in the respective |NCS| release at its release date.
      Therefore, to see the official version of the release notes for a specific |NCS| release, switch to the documentation for the corresponding |NCS| version using the selector in the upper left-hand corner.
 
 .. toctree::
    :maxdepth: 1
    :caption: Subpages:
 
+   releases/release-notes-changelog
```
Comment (umapraseeda) [inline doc/nrf/release_notes.rst:23]: Also, we need to remove the :orphan: tag from release-notes-changelog

**Review by greg-fer — APPROVED** [review]
Changelog was cleaned in the release notes PR, so this looks good to go for me.

File: doc/nrf/release_notes.rst (hunk 2 above)
Comment (b-gent) [reply doc/nrf/release_notes.rst:23]: Done now, thought I did this but must have not committed

File: doc/nrf/release_notes.rst (hunk 1 above)
Comment (b-gent) [reply doc/nrf/release_notes.rst]: done

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -131,21 +129,21 @@ Zephyr
...
-For a complete list of |NCS| specific commits, run:
+For the list of nRF Connect SDK specific commits, including commits cherry-picked from upstream, run:
 
 .. code-block:: none
 
-   git log --oneline manifest-rev ^3f82656
+   git log --oneline manifest-rev ^45ef0d2
 
-The current |NCS| main branch is based on the Zephyr v2.7 development branch.
+The current nRF Connect SDK main branch is based on revision ``45ef0d2`` of Zephyr, which is located between v2.7.0 and v3.0.0 of Zephyr.
```
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-changelog.rst:146]:
```suggestion
The current |NCS| main branch is based on revision ``45ef0d2`` of Zephyr, which is located between v2.7.0 and v3.0.0 of Zephyr.
```

**Review by umapraseeda — APPROVED** [review]
Just a minor fix..LGTM otherwise.

**Comment by umapraseeda** [comment]
Also, do we need the mcumgr section in the 1.9.99 changelog as we had removed it  in the release notes for 1.9, @carlescufi or @shantha-14 ?

**Comment by carlescufi** [comment]
> Also, do we need the mcumgr section in the 1.9.99 changelog as we had removed it in the release notes for 1.9, @carlescufi or @shantha-14 ?

No, mcumgr is no longer a separate project but rather part of Zephyr, so this section is no longer required.

**Comment by umapraseeda** [comment]
> > Also, do we need the mcumgr section in the 1.9.99 changelog as we had removed it in the release notes for 1.9, @carlescufi or @shantha-14 ?
> 
> No, mcumgr is no longer a separate project but rather part of Zephyr, so this section is no longer required.

Could you remove that section @b-gent ?

**Review by carlescufi — APPROVED** [review]

**Review by shantha-14 — APPROVED** [review]

**Comment by b-gent** [comment]
> > > Also, do we need the mcumgr section in the 1.9.99 changelog as we had removed it in the release notes for 1.9, @carlescufi or @shantha-14 ?
> > 
> > 
> > No, mcumgr is no longer a separate project but rather part of Zephyr, so this section is no longer required.
> 
> Could you remove that section @b-gent ?

removed

**Comment by grochu** [comment]
Hi @umapraseeda, I posted this PR: https://github.com/nrfconnect/sdk-nrf/pull/6935 to fix release notes highlights - I overlooked an issue during the Release Notes review. If you prefer, the change can be incorporated to this PR. I think it is important to fix this statement to avoid misleading information.

**Pull request merged**